exp-player/addon/components/exp-player.js

```diff
@@ -50,12 +55,33 @@ export default Ember.Component.extend({
         return ctx;
     }),
     actions: {
+        saveFrame(frameId, frameData) {
+            // Save the data from a completed frame to the session data item
+            var expData = this.get('expData');
+            expData[frameId] = frameData;
+            this.set('expData', expData);
+        },
+        saveSession() {
+            // Construct payload and send to server
+            var payload = {
+                profileId: null,  // TODO: fetch this from a page session with user id
```

Comment:
We can probably use ember-simple-auth to write this to the session, then fetch it out like:

https://github.com/CenterForOpenScience/experimenter/pull/9/files#diff-969098c30cbcdabdb6db38a00451006cR6